INSTALL.md

```diff
@@ -56,151 +62,158 @@ sudo dnf install ruby ruby-devel rubygem-rdoc rubygem-bundler rubygems \
 
 If you didn't already have PostgreSQL installed then create a PostgreSQL instance and start the server:
 
-```
+```bash
 sudo postgresql-setup initdb
```

Comment:
I'm guessing maybe I put this in at some point but `postgresql-setup` is Fedora specific I think. The equivalent on Debian/Ubuntu would be something with `pg_createcluster` though unlike Fedora I think installing postgres usually creates a default cluster there.

Reply:
Yes, installing on Debian or Ubuntu creates a cluster with the standard setup.

Reply:
I still needed to start postgres with `systemctl` when I installed it on Ubuntu 24.04. I don't remember having to do that on 22.04.